### PR TITLE
Add tooltip to ownership card tiles

### PR DESCRIPTION
**Pull request by jessebye**
## Hey, I just made a Pull Request!

Adds a tooltip to the ownership card tiles listing entity names in each category:

Screen Shot 2021-06-16 at 1 27 56 PM

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx [hunk 1]
```diff
@@ -94,39 +95,49 @@ const useStyles = makeStyles((theme: BackstageTheme) =>
...
-      alignItems="center"
-    >
-      <Typography className={classes.bold} variant="h6">
-        {counter}
-      </Typography>
-      <Typography className={classes.bold} variant="h6">
-        {name}
-      </Typography>
-    </Box>
+    <Tooltip title={`${entities.join(', ')}`} arrow>
```
Comment (Rugvip) [inline plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx:126]: I'm thinking it's best to add some sensible max count at which point it falls back to filling in with `...`, unless the tooltip already handles that

File: .changeset/bright-wolves-watch.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': minor
```
Comment (Rugvip) [inline .changeset/bright-wolves-watch.md:2]: Not a breaking change right? So can make this `patch`

**Review by Rugvip — COMMENTED** [review]
Thank you! makes sense imo

Seems like there's a couple of bonus commits here that you need to get rid of though 😅

**Comment by jessebye** [comment]
I'm going to open another branch for this as I made a mess of this one 😬

**Pull request closed without merging**